content/guides/requesting-payment.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Requesting Payment
+nav:
+  path: /reference/merchant-integrations
```

Comment:
I have changed the format of the nav path.

I couldn't remove the `toKebabCase` function though because we still need a format of the nav path that can be compared to the `to` property on nav items. This is so we can highlight the parent navigation items. E.g.

Nav path: `References/Merchant Integrations`

So the `References` section and the `Merchant Integrations` section need to be highlighted. This is done by using the `to` property on the navigation item which are `/references` and `/references/merchant-integrations` respectively.

We need to use a `startsWith` syntax to make sure both sections are highlighted so we need to transform the nav path into a route-style path --> `References/MerchantIntegrations --> /references/merchant-integrations`